src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Internal/MsQuicApi.cs

```diff
@@ -0,0 +1,257 @@
+                    NativeRegistration.ConnectionClose);
+            ConnectionShutdownDelegate =
+                Marshal.GetDelegateForFunctionPointer<MsQuicNativeMethods.ConnectionShutdownDelegate>(
+                    NativeRegistration.ConnectionShutdown);
+            ConnectionStartDelegate =
+                Marshal.GetDelegateForFunctionPointer<MsQuicNativeMethods.ConnectionStartDelegate>(
+                    NativeRegistration.ConnectionStart);
+
+            StreamOpenDelegate =
+                Marshal.GetDelegateForFunctionPointer<MsQuicNativeMethods.StreamOpenDelegate>(
+                    NativeRegistration.StreamOpen);
+            StreamCloseDelegate =
+                Marshal.GetDelegateForFunctionPointer<MsQuicNativeMethods.StreamCloseDelegate>(
+                    NativeRegistration.StreamClose);
+            StreamStartDelegate =
+                Marshal.GetDelegateForFunctionPointer<MsQuicNativeMethods.StreamStartDelegate>(
+                   NativeRegistration.StreamStart);
+            StreamShutdownDelegate =
+                Marshal.GetDelegateForFunctionPointer<MsQuicNativeMethods.StreamShutdownDelegate>(
+                    NativeRegistration.StreamShutdown);
+            StreamSendDelegate =
+                Marshal.GetDelegateForFunctionPointer<MsQuicNativeMethods.StreamSendDelegate>(
+                    NativeRegistration.StreamSend);
+            StreamReceiveCompleteDelegate =
+                Marshal.GetDelegateForFunctionPointer<MsQuicNativeMethods.StreamReceiveCompleteDelegate>(
+                    NativeRegistration.StreamReceiveComplete);
+            StreamReceiveSetEnabledDelegate =
+                Marshal.GetDelegateForFunctionPointer<MsQuicNativeMethods.StreamReceiveSetEnabledDelegate>(
+                    NativeRegistration.StreamReceiveSetEnabled);
+            SetContextDelegate =
+                Marshal.GetDelegateForFunctionPointer<MsQuicNativeMethods.SetContextDelegate>(
+                    NativeRegistration.SetContext);
+            GetContextDelegate =
+                Marshal.GetDelegateForFunctionPointer<MsQuicNativeMethods.GetContextDelegate>(
+                    NativeRegistration.GetContext);
+            SetCallbackHandlerDelegate =
+                Marshal.GetDelegateForFunctionPointer<MsQuicNativeMethods.SetCallbackHandlerDelegate>(
+                    NativeRegistration.SetCallbackHandler);
+
+            SetParamDelegate =
+                Marshal.GetDelegateForFunctionPointer<MsQuicNativeMethods.SetParamDelegate>(
+                    NativeRegistration.SetParam);
+            GetParamDelegate =
+                Marshal.GetDelegateForFunctionPointer<MsQuicNativeMethods.GetParamDelegate>(
+                    NativeRegistration.GetParam);
+        }
+
+        internal MsQuicNativeMethods.NativeApi NativeRegistration { get; private set; }
+
+        internal MsQuicNativeMethods.RegistrationOpenDelegate RegistrationOpenDelegate { get; private set; }
```

Comment:
These can be readonly fields. No need to pay the extra overhead of properties for repetitive implementation details like this.